### PR TITLE
Opinion Card Styles

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds styling for an opinion card

Screenshot 2019-11-25 at 15 48 44

## Why?
Parity

## Link to supporting Trello card
https://trello.com/c/CYwWtLob/889-style-cards-for-opinion

### CODE REVIEW

File: src/web/components/Card/components/AvatarContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export const AvatarContainer = ({ children }: Props) => {
+    return (
+        <div
+            className={css`
+                display: flex;
+                flex-direction: row-reverse;
+
+                margin-right: 10px;
+                margin-top: 50px;
```
Comment (gtrufitt) [inline src/web/components/Card/components/AvatarContainer.tsx]: Is this magic? Not sure what the 50px relates to exactly

File: src/web/components/Card/components/AvatarSize.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (gtrufitt) [inline src/web/components/Card/components/AvatarSize.tsx]: As both avatarContainer and avatarSize are in the `Card` folder, do they need to be separated? I can see them being separate if the container is used elsewhere and the size is somewhere else but I wonder whether they're in the right place/could be combined/could have props passed to size so we can use everywhere avatar is?

File: src/web/components/Card/components/AvatarContainer.tsx (hunk 1 above)
Comment (oliverlloyd) [reply src/web/components/Card/components/AvatarContainer.tsx]: Somewhat magic. It's the space that lives above an avatar when it's inside a card

File: src/web/components/Card/components/AvatarSize.tsx (hunk 2 above)
Comment (oliverlloyd) [reply src/web/components/Card/components/AvatarSize.tsx]: As discussed irl, it's good to keep `Avatar` simple and use wrappers to add sizing and margins. In this case we have a Card specific wrapper.

I merged `ArticleSize` into `ArticleContainer`.

**Pull request merged**